x-pack/plugins/security_solution/public/timelines/components/timeline/properties/helpers.tsx

```diff
@@ -45,14 +46,18 @@ const NotesButtonContainer = styled(EuiFlexGroup)`
 
 const SmallNotesButton = React.memo<SmallNotesButtonProps>(
   ({ ariaLabel = i18n.NOTES, isDisabled, toggleShowNotes, timelineType, eventId, notesCount }) => {
+    const { telemetry } = useKibana().services;
     const isTemplate = timelineType === TimelineType.template;
     const onClick = useCallback(() => {
       if (eventId != null) {
+        telemetry.reportTimelinesAddDocumentNote({
```

Comment:
Should this be reported when `Add note` is clicked.

I think either that or we should have 2 events : 

1. noteAddAttempted ( maybe needs a better name ) or it can simply be a click event ?
2. noteAdded